code/ModelAdmin.php

```diff
@@ -675,8 +678,7 @@ public function import(array $data, Form $form): HTTPResponse
             $results = $loader->load($_FILES['_CsvFile']['tmp_name']);
         } catch (HTTPResponse_Exception $e) {
             $form->sessionMessage($e->getMessage(), ValidationResult::TYPE_ERROR);
-            $this->redirectBack();
-            return false;
+            return $this->redirectBack();
```

Comment:
`false` is not a valid return type for this method in Silverstripe CMS 5.
It causes a behat failure in various scenarios of the "Import in GridField" behat feature:
> Uncaught TypeError: SilverStripe\Admin\ModelAdmin::import(): Return value must be of type SilverStripe\Control\HTTPResponse, bool returned